setup.py

```diff
@@ -114,9 +114,8 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
-        'Development Status :: 3 - Alpha',
+        'Programming Language :: Python :: Implementation :: PyPy',
```

Comment:
Should we also add `Programming Language :: Python :: Implementation :: CPython` in addition to `PyPy`?